CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+# Contributing guide
+
+Welcome to Percona Backup for MongoDB!
+
+1. [Prerequisites](#prerequisites)
+2. [Submitting a pull request](#submitting-a-pull-request)
+3. [Contributing to documentation](#contributing-to-documentation)
+
+We're glad that you would like to become a Percona community member and participate in keeping open source open.  
+
+Percona Backup for MongoDB is a distributed, low-impact solution for achieving consistent backups of MongoDB sharded clusters and replica sets.
+
+You can contribute in one of the following ways:
+
+1. Reach us on our [Forums](https://forums.percona.com) and [Discord]([https://discord.gg/mQEyGPkNbR](https://discord.gg/mQEyGPkNbR)).
+2. [Submit a bug report or a feature request](https://github.com/percona/percona-backup-mongodb/blob/main/README.md) 
+3. Submit a pull request (PR) with the code patch
+4. Contribute to documentation 
+
+## Prerequisites
+
+Before submitting code contributions, we ask you to complete the following prerequisites.
+
+### 1. Sign the CLA
+
+Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/&lt;linktoCLA>) (CLA). You can do this in on click using your GitHub account.
+
+**Note**:  You can sign it later, when submitting your first pull request. The CLA assistant validates the PR and asks you to sign the CLA to proceed.
+
+### 2. Code of Conduct
+
+Please make sure to read and agree to our [Code of Conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
+
+## Submitting a pull request
+
+All bug reports, enhancements and feature requests are tracked in [Jira issue tracker](https://jira.percona.com/projects/PBM). Though not mandatory, we encourage you to first check for a bug report among Jira issues and in the PR list: perhaps the bug has already been addressed. 
+
+For feature requests and enhancements, we do ask you to create a Jira issue, describe your idea and discuss the design with us. This way we align your ideas with our vision for the product development.
+
+If the bug hasn’t been reported / addressed, or we’ve agreed on the enhancement implementation with you, do the following:
+
+1. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) this repository
+2. Clone this repository on your machine and sync it with upstream. 
```

Comment:
```suggestion
2. Clone this repository on your machine
```

At the moment you do a clone it will be synchronized with the latest upstream commits.